wiki/Guides/Cropping_with_Complex_Backgrounds/tr.md

```diff
@@ -0,0 +1,39 @@
+# Karışık arkaplanlara sahip görseller nasıl kırpılır
+
+## Adım \#1
+
+Öncelikle, bir resim seçmeniz gerekli. Ben burada gördüğünüz görseli size sunacağım. Ayrıca bunu sizlere GIMP kullanarak nasıl yapacağınızı göstereceğim. (Diğer progamlar ile bunun arasında çok fazla sorun çıkmaması lazım.) Bu yöntem için serbest seçim aletine yani Kement aletine ihtiyacımız var.
+
+![Öğretici Görsel 1](img/CCB-1.png "Tutorial Image 1")
```

Comment:
```suggestion
![Öğretici Görsel 1](img/CCB-1.png "Öğretici Görsel 1")
```
The title text (the one in quotes on the right) should also be translated. This is also the image caption on the website.